utils/environment_linux.yml

```diff
@@ -164,4 +163,6 @@ dependencies:
   - zipp=0.6.0=py_0
   - zlib=1.2.11=h7b6447c_3
   - zstd=1.3.7=h0b5b093_0
+  - pip:
+    - sphinx_rtd_theme==0.5.0rc1
```

Comment:
Can we use a "non-`pip`" `sphinx_rtd_theme`? I believe it made things easier in the Travis/Docker world...

Reply:
Done! I didn't know this was the reason why all the pip packages disappeared at some point